webpack.config.js

```diff
@@ -57,5 +58,12 @@ module.exports = {
         use: ['style-loader', 'css-loader'],
       },
     ]
-  }
+  },
+  performance: {
+    assetFilter: (filename) => {
```

Comment:
Better use the default ammended with special case.
`return !((/\.map$/.test(filename)) || (filename === 'swagger.js'))`

Reply:
Ok, committed a slight variation of this.